src/WebJobs.Script/Diagnostics/Extensions/ScriptHostLoggerExtension.cs

```diff
@@ -17,17 +16,11 @@ internal static class ScriptHostLoggerExtension
             new EventId(400, nameof(HostIdIsSet)),
             "Host id explicitly set in configuration. This is not a recommended configuration and may lead to unexpected behavior.");
 
-        private static readonly Action<ILogger, string, Exception> _startingHost =
-            LoggerMessage.Define<string>(
-            LogLevel.Information,
-            new EventId(401, nameof(StartingHost)),
-            "{message}");
-
-        private static readonly Action<ILogger, string, Exception> _functionsErrors =
-            LoggerMessage.Define<string>(
+        private static readonly Action<ILogger, string, string, Exception> _functionError =
+            LoggerMessage.Define<string, string>(
             LogLevel.Error,
-            new EventId(402, nameof(FunctionsErrors)),
-            "{message}");
+            new EventId(402, nameof(FunctionError)),
+            "The '{functionName}' function is in error: {errorMessage}");
```

Comment:
Note that this log changed. It now will write out each individual function name with its error, which allows us to use `functionName` for better querying in Kusto (and elsewhere).